.github/workflows/backend-ci.yml

```
@@ -41,10 +41,10 @@ jobs:
       pull-requests: read # Or 'write' if reviewdog needs to create PR review comments directly
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)